pkgs/development/python-modules/jaxlib/default.nix

```diff
@@ -195,7 +195,12 @@ let
     '';
 
     bazelRunTarget = "//jaxlib/tools:build_wheel";
-    runTargetFlags = [ "--output_path=$out" "--cpu=${arch}" ];
+    runTargetFlags = [
+      "--output_path=$out"
+      "--cpu=${arch}"
+      # This has no impact whatsoever...
+      "--jaxlib_git_hash='12345678'"
```

Comment:
can we just skip it then? i'd prefer to not insert magical numbers if we can avoid

or why not use `src.rev`?

Reply:
Alternatives considered: `0000000000000000000000000000000000000000` 😆